### PR TITLE
add ITN favicon and fix links

### LINKED ISSUE #55: broken links
noticed broken link to my course after I changed the link https://jhudatascience.org/Data_Management_for_Cancer_Research/data-privacy.html in 4 in at least 2 places

### PR DESCRIPTION
**Pull request by carriewright11**
Modifying index file to update the favicon. I noticed this when I was showing your course of to someone today ;)

Also fixing links that I found to be out of date- this resolves #53  and resolves #55  - github changed several of their links and my course has a different name, and Sarah isn't at JHU anymore, and the license link has a parenthesis that was throwing it off - so I found a new link to describe the license. I think our recent OTTR updates should fix that but did this just in case. I also noticed that the badge was for the template, so I made it for this repo.

Also updating the README which seems to still be for the first course of the series in a couple of spots.

### CODE REVIEW

File: 05-code-review-author.Rmd [hunk 1]
```diff
@@ -115,8 +115,8 @@ ottrpal::include_slide("https://docs.google.com/presentation/d/1IJ_uFxJud7OdIAr6
 
 1. Create a new branch as we described in the previous chapter.
 2. In your local repository, create a folder called `.github`
-3. Download this [pull request template file](https://raw.githubusercontent.com/jhudsl/Adv_Reproducibility_in_Cancer_Informatics/main/resources/PULL_REQUEST_TEMPLATE.md) to get you started.
-4. Feel free to edit this file to your own needs and add it to the `.github` folder.
+3. Cpy and past this [pull request template file](https://raw.githubusercontent.com/jhudsl/Adv_Reproducibility_in_Cancer_Informatics/main/.github/PULL_REQUEST_TEMPLATE.md) to a new text file and save it as a `.md` to get started.
```
Comment (avahoffman) [inline 05-code-review-author.Rmd:118]:
```suggestion
3. Copy and paste this [pull request template file](https://raw.githubusercontent.com/jhudsl/Adv_Reproducibility_in_Cancer_Informatics/main/.github/PULL_REQUEST_TEMPLATE.md) to a new text file and save it as a `.md` to get started.
```

**Review by avahoffman — APPROVED** [review]
Looks good! A few minor notes :)

Funky formatting of the author:
image

Are the dashes intentional here?
image

File: book.bib [hunk 2]
```diff
@@ -384,26 +384,24 @@ @misc{GitHub2021b
 	year = {2021},
 }
 
-@misc{GitHub2021c,
+@misc{GitHub2022c,
```
Comment (carriewright11) [inline book.bib:387]:
```suggestion
@online{GitHub2022c,
```

File: book.bib [hunk 3]
```diff
@@ -384,26 +384,24 @@ @misc{GitHub2021b
...
 	language = {en},
-	urldate = {2021-11-30},
-	year = {2021},
-	journal = {GitHub Learning Lab},
+	urldate = {2022-12-12},
+	year = {2022}
 }
 
-@misc{GitHub2021d,
+@misc{GitHub2022d,
```
Comment (carriewright11) [inline book.bib:397]:
```suggestion
@online{GitHub2022d,
```

File: book.bib [hunk 4]
```diff
@@ -394,7 +394,7 @@ @online{GitHub2022c
 	year = {2022}
 }
 
-@misc{GitHub2022d,
+@online{GitHub2022d,
 	author = {The GitHub Training Team},
```
Comment (carriewright11) [inline book.bib:398]:
```suggestion
```

File: book.bib [hunk 5]
```diff
@@ -384,26 +384,24 @@ @misc{GitHub2021b
 	year = {2021},
 }
 
-@misc{GitHub2021c,
+@online{GitHub2022c,
 	author = {The GitHub Training Team},
```
Comment (carriewright11) [inline book.bib:388]:
```suggestion
```

**Pull request merged**